packages/data-explorer/tests/unit/mappers/mapperUtils.spec.ts

```diff
@@ -0,0 +1,103 @@
+}))
+
+describe('getCategoricals', () => {
+  it('Will filter out CATEGORICAL meta data ', async () => {
+    const categoricals: MetaDataAttribute[] = getCategoricals(meta.attributes)
+    expect(JSON.stringify(categoricals)).toEqual('[{"href":"/api/v2/test/meta/country","fieldType":"CATEGORICAL","name":"country","label":"country","attributes":[],"auto":false,"nillable":false,"readOnly":true,"labelAttribute":false,"unique":true,"visible":true,"lookupAttribute":true,"isAggregatable":false,"refEntity":{"name":"countries","idAttribute":"id","labelAttribute":"label","hrefCollection":"/api/v2/countries","href":"/api/v2/countries"}},{"href":"/api/v2/test/meta/age_groups","fieldType":"CATEGORICAL_MREF","name":"age_groups","label":"age_groups","attributes":[],"auto":false,"nillable":false,"readOnly":true,"labelAttribute":false,"unique":true,"visible":true,"lookupAttribute":true,"isAggregatable":false,"refEntity":{"name":"age_groups","idAttribute":"id","labelAttribute":"label","hrefCollection":"/api/v2/age_groups","href":"/api/v2/age_groups"}}]')
+  })
+})
+
+describe('uriGenerator', () => {
+  const uri = '/api/v2/tablename'
+  const id = 'idAttr'
+  const label = 'labelAttr'
+
+  it('should generate an uri without query when there is no search string',
+    () => {
+      const search = null
+      const result = generateUri(uri, id, label, search)
+      expect(result).toEqual(uri)
+    })
+
+  it('should generate an uri with like query when there is a search string',
+    () => {
+      const search = 'blaat'
+      const result = generateUri(uri, id, label, search)
+      const expected = `${uri}?q=${id}=like=${search},${label}=like=${search}`
+      expect(result).toEqual(expected)
```

Comment:
suggest writing out the expected string result, this makes it easier to follow along and spot mistakes in the test